modules/ROOT/pages/create-community.adoc

```diff
@@ -258,7 +258,7 @@ These permissions reflect the current API Community Manager package version. If
 Configure the settings with one of these options:
 
 * Configure sharing settings for members and guests automatically using the default settings specified by the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
-* Configure sharing settings for members and guests manually using custom settings applied in a series of steps.
+* Configure sharing settings for members and guests manually using custom settings applied in the following steps.
 
 Configure sharing settings for members:
```

Comment:
```suggestion
* Configure sharing settings for members and guests manually using custom settings applied in the following steps:
```

Reply:
This change would make the instructions incorrect, because the reader must follow the instructions that begin "Configure sharing settings for members:" and also follow the instructions immediately after it that begin "Configure sharing settings for guests:".